packages/serializer/package.json

```diff
@@ -25,12 +25,14 @@
     "@babel/core": "^7.4.4",
     "@babel/helper-plugin-utils": "^7.0.0",
     "@babel/plugin-syntax-jsx": "^7.2.0",
+    "@mdx-js/mdx": "^1.0.16",
```

Comment:
Are you using mdx anywhere? Might've missed it

Reply:
Ah, nah I nuked my usage of it for a simpler (hackier version).